kazoo/tests/test_client.py

```diff
@@ -775,6 +775,19 @@ def test_create_exists(self):
         path = client.create("/1")
         self.assertRaises(NodeExistsError, client.create, path)
 
+    def test_create_stat(self):
+        if TRAVIS_ZK_VERSION:
+            version = TRAVIS_ZK_VERSION
+        else:
+            version = self.client.server_version()
+        if not version or version < (3, 5):
+            raise SkipTest("Must use Zookeeper 3.5 or above")
```

Comment:
I haven't looked deeply at this code, so this is more of a side-comment, but long term it'd probably be more convenient to use `pytest`'s `skipif` decorator for things like this. That's a much bigger conversation about migrating the test suite to `pytest` and this is probably fine as-is if this pattern is consistently used elsewhere in Kazoo.

Reply:
I agree.

Reply:
Only question I have is would a decorate be able to access `client.server_version()` ? It would require the client to be connected, no?

In either case, I agree with the move to pytest. Created #589.